tests/src/test/scala/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -604,7 +597,13 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
                 response should be(activation.withoutLogs.toExtendedJson)
             }
 
-            // repeat invoke, get only result back
+            // repeat invoke, get only result back (result option implies blocking invocation)
```

Comment:
Looks like this file should be unchanged as these tests communicate directly with the backend. Currently the backend does not automatically convert the result option to a blocking invocation.